app/views/users/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_tag users_path do %>
+  <p>Name: <%= text_field_tag :name, @user.name %></p>
+  <p>Address: <%= text_field_tag :address, @user.address %></p>
+  <p>City: <%= text_field_tag :city, @user.city %></p>
+  <p>State: <%= text_field_tag :state, @user.state %></p>
+  <p>Zip: <%= text_field_tag :zip, @user.zip %></p>
+  <p>Email: <%= text_field_tag :email %></p>
+  <p>Password: <%= password_field_tag :password %></p>
+  <p>Confirm Password: <%= password_field_tag :password_confirmation %></p>
+  <p><%= submit_tag "Create User" %></p>
+<%end %>
```

Comment:
We will want to move part of this to a partial since it is very similar to when we need to edit a user profile.